api/server/sdk/cloud_backup.go

```diff
@@ -37,22 +37,28 @@ func (s *CloudBackupServer) Create(
 	req *api.SdkCloudBackupCreateRequest,
 ) (*api.SdkCloudBackupCreateResponse, error) {
 
-	if len(req.GetVolumeId()) == 0 {
+	if len(req.GetName()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Must supply a name")
+	} else if len(req.GetVolumeId()) == 0 {
 		return nil, status.Error(codes.InvalidArgument, "Must supply a volume id")
 	} else if len(req.GetCredentialId()) == 0 {
 		return nil, status.Error(codes.InvalidArgument, "Must supply credential uuid")
 	}
 
 	// Create the backup
-	if err := s.driver.CloudBackupCreate(&api.CloudBackupCreateRequest{
+	r, err := s.driver.CloudBackupCreate(&api.CloudBackupCreateRequest{
```

Comment:
Is idempotency checked by this function?

Reply:
yes